README.md

```diff
@@ -1,19 +1,53 @@
 # ImageDePHI
+ImageDePHI is an application that can be used to redact PHI from whole slide images.
 
-This project has been funded in whole or in part with Federal funds from the National Cancer Institute, National Institutes of Health, Department of Health and Human Services, under Contract No. 75N91022C00033
 
-## To Run as a CLI
+# Installation
+ImageDePHI is under active development and as such the best way to test the latest features is to download the most current artifact.
 
-`pip install -r requirements.txt`
 
-`python imagedephi.py <directory of input images> <directory to store redacted images>`
+[ImageDePHI Artifacts](https://github.com/DigitalSlideArchive/ImageDePHI/actions/workflows/ci.yml?query=branch%3Amain)
 
+Select the most recent run:
+![ci run](imagedephi/assets/ci_run.png)
 
-## Development
+Then select the appropriate artifact for your OS:
+![artifacts](imagedephi/assets/artifacts_list.png)
```

Comment:
I think we can have a change very soon which just publishes binaries directly with release. Asking people to go into CI runs is too burdensome just to download an application.

Reply:
Once we have a release I will update this to link to that instead